pkg/transformer/openshift/openshift_test.go

```diff
@@ -17,13 +17,13 @@ limitations under the License.
 package openshift
 
 import (
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
can we remain consistent with `openshit.go` and use the alias for k8s api import as kapi ?